NumberValidator/Helpers/ExtensionMethods.cs

```diff
@@ -22,5 +22,17 @@ public static bool IsDigits(this string input)
 
             return digitsRegularExpression.Match(input).Success;
         }
+
+        public static string RemoveSpace(this string input)
+        {
+            if (string.IsNullOrEmpty(input))
+            {
+                return string.Empty;
+            }
+
+            var cleanedInput = Regex.Replace(input, @"[\s-]", "");
+
+            return cleanedInput.ToUpper().Trim();
```

Comment:
`RemoveSpace` should just do what it is saying i.e. `RemoveSpace` and not to `ToUpper` and `Trim`

Reply:
ok sir i will update the changes

Reply:
i changed the function name